recipes/jsonpath-ng/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "jsonpath-ng" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b1fc75b877e9b2f46845a455fbdcfb0f0d9c727c45c19a745d02db620a9ef0be
+
+build:
+  number: 0
```

Comment:
This package can be noarch. You need to add console_scripts

Reply:
Thanks, Have created [PR](https://github.com/conda-forge/jsonpath-ng-feedstock/pull/2) to address this